Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -1,47 +1,24 @@
-﻿using Apsitvarkom.Models.DTO;
+﻿using Apsitvarkom.DataAccess;
+using Apsitvarkom.Models.DTO;
 using Microsoft.AspNetCore.Mvc;
-using System.Globalization;
 
 namespace Apsitvarkom.Api.Controllers;
 
 [ApiController]
 [Route("/api/[controller]")]
 public class PollutedLocationController : ControllerBase
 {
+    private readonly IPollutedLocationDTORepository _repository;
+
+    public PollutedLocationController(IPollutedLocationDTORepository repository)
+    {
+        _repository = repository;
+    }
+
     [HttpGet("All")]
```

Comment:
Explicit swagger documentation with `ProducesResponseType` attributes would be great IMO:
https://medium.com/@mohammed0hamdan/using-producesresponsetype-to-write-a-better-web-api-actions-in-net-core-18e080c9bf00

This action probably now returns 200 on success and 500 on exception

Reply:
Nice one, didn't know about this.